csuntechlab/iSTART
# STR-150 -- FE: Implement Card Flip Informational Slides #13, #18, #25, #31

# Description
Implemented card flip slides
  
# Testing
1. Login as `cyndee.bellamy` or `april.feldman`.
2. Visit alcohol module
3. Ensure the card flip slides are implemented to specification.
4. Check to make sure the text makes sense in our application.
